src/lib/components/Navbar/NavbarLink.tsx

```diff
@@ -11,28 +12,28 @@ export interface FlowbiteNavbarLinkTheme {
   disabled: FlowbiteBoolean;
 }
 
-export interface NavbarLinkProps extends PropsWithChildren<ComponentProps<'a'>> {
+export interface NavbarLinkProps extends PropsWithChildren<ComponentProps<'a'>>, Partial<Pick<LinkProps, 'to'>> {
```

Comment:
is `to` optional here?

Reply:
yes, you can use both `a` tag and `Link` component from react-router-dom.